src/test/java/com/purbon/kafka/topology/AccessControlManagerTest.java

```diff
@@ -10,48 +10,26 @@
 import static org.mockito.Matchers.anyString;
 import static org.mockito.Matchers.eq;
 import static org.mockito.Matchers.refEq;
-import static org.mockito.Mockito.doNothing;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.*;
 
 import com.purbon.kafka.topology.actions.Action;
 import com.purbon.kafka.topology.api.adminclient.AclBuilder;
-import com.purbon.kafka.topology.model.Component;
+import com.purbon.kafka.topology.model.*;
```

Comment:
actually, this is an interesting one, adding * or single classes :-P I guess I don't care that much, but not sure what is generally cleaner.

Reply:
I used to be a "no wildcard imports"-guy, but now I just leave it to the IDE (in my case IntelliJ IDEA) to decide. I never look at them anyway.

Reply:
This particular case does not look like something IDEA would do. Probably my rebasing.

Reply:
Honestly, I am not that strict I was more looking for learning your ways. I like the refactoring, thanks man!

Reply:
Thank you! My ways are generally "keep it simple". :-)